app/services/assignment_service.py

```diff
@@ -307,34 +315,30 @@ def get_adjusted_due_date(self) -> datetime | None:
 
         return self.assignment.due_date + deferred_time + extra_time + self.student.base_extra_time
 
-    def get_is_available(self) -> bool:
-        if not self.assignment.is_created: return False
-
+    def _get_is_available(self) -> bool:
         current_timestamp = self.session.scalar(func.current_timestamp())
         return current_timestamp >= self.get_adjusted_available_date()
     
-    def get_is_closed(self) -> bool:
-        if not self.assignment.is_created: return False
-
+    def _get_is_closed(self) -> bool:
         current_timestamp = self.session.scalar(func.current_timestamp())
         return current_timestamp > self.get_adjusted_due_date()
```

Comment:
same issue here as in instructorassignmentservice's methods. comparing a nullish to an int